src/test/java/roomescape/member/MemberControllerTest.java

```diff
@@ -0,0 +1,93 @@
+package roomescape.member;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import roomescape.member.dto.LoginMemberRequestDto;
+import roomescape.member.dto.MemberResponseDto;
+
+import static org.assertj.core.api.Assertions.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+class MemberControllerTest {
+
+    private static final String EMAIL = "json@email.com";
+    private static final String PASSOWORD = "1234";
+
+    @DisplayName("로그인을 하면 토큰을 응답받는다.")
+    @Test
+    void basicLogin(){
+
+        final LoginMemberRequestDto loginMemberRequestDto = new LoginMemberRequestDto(EMAIL, PASSOWORD);
+        var response = RestAssured
+                .given().log().all()
+                .body(loginMemberRequestDto)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .accept(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/login")
+                .then().log().all().extract();
+
+        assertThat(response.cookie("token")).isNotEmpty();
```

Comment:
```suggestion
        RestAssured
            .given().log().all()
            .body(loginMemberRequestDto)
            .contentType(MediaType.APPLICATION_JSON_VALUE)
            .accept(MediaType.APPLICATION_JSON_VALUE)
            .when().post("/login")
            .then().log().all()
            .assertThat()
            .cookie("token", notNullValue());
```

RestAssured 는 생각보다 훨씬 많은 기능을 제공해요.
공식 문서도 그리 어렵지 않으니 한 번 확인해보고, 이렇게 응답이나 쿠키를 추출하여 검증하는 방식 외에
바로 검증하는 방법도 도전해보면 재밌을 것 같아요!